app.py

```diff
@@ -8,14 +8,16 @@
 
 @app.route('/')
 def hello_world():
-    app.logger.info("Hello, World!")
-    return 'Hello, World!'
+    message = "Hello, World!"
+    app.logger.info(f"{message}")
+    return message
 
 
 @app.route('/error')
 def error():
-    app.logger.error("I'm an error")
-    return "I'm an error"
+    error_message = "I'm an error"
+    app.logger.error(f"{error_message}")
+    return error_message
```

Comment:
Chyba nie do końca o to mi chodziło, być może to ja tu czegoś nie rozumiem, ale
jak masz metodę ogólną `error()`, to nie chcemy móc przekazywać wiadomości do tej metody? 
   `def error(error_message: str = "I'm an error")`

Celowo jest to pominięte?